filestore/test/test_commands.py

```diff
@@ -10,21 +10,19 @@
                            register_handler, deregister_handler,
                            bulk_insert_datum)
 from filestore.core import DatumNotFound
-from filestore.utils.testing import fs_setup, fs_teardown
 from numpy.testing import assert_array_equal
-from nose.tools import assert_raises
 
-from .t_utils import SynHandlerMod
+from .utils import SynHandlerMod,  fs_setup, fs_teardown
```

Comment:
I tried to not bikeshed this, but there is an extra space here and it is bothering me...